client/src/components/Survey.jsx

```diff
@@ -179,8 +173,12 @@ class Survey extends React.Component {
           </div>
           <input
```

Comment:
these two elements (input and button) are neither in a col div nor a row div. probably you should put them in a row div and then in a col div. also, can we just lose the "clear form" button? it's kinda meh